### ISSUE TITLE
Menus with sub-sub menus on hover

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_microsites_base#193]
@markconroy 
I think we still need to do some work on the menu on hover. 

If we have a menu item with multiple children and some grandchildren eg

![Screenshot from 2022-12-08 10-54-48](https://user-images.githubusercontent.com/3852805/206429283-eabf1e5a-b1da-4609-a76e-ecc4192cc8f8.png)

then when we hover over the first child we can hover to its child, but the menu disappears before we can hover over the second child. 

Hard to describe, let's discuss.

**Comment by markconroy** [comment]
I've just noticed this too. Not sure why we are having this, but I'm sure we'll figure it out.

**Comment by markconroy** [comment]
From what I've been reading recently (and to make sure we have a similar experience for all users) we should not set hover states on menus, and instead make people explicitly click the 'expand' button, like keyboard users do.

That takes away a lot of the complexity of hover vs focus (and trying to keep them in tandem if someone clicks an expand button while hovering etc) and ensures all users perform the same actions.

I've a comment here - https://github.com/localgovdrupal/localgov_microsites_base/issues/219 - about moving to the Annertech disclosure library. Let's chat about that and see if we can solve a lot of the menu issues in one fell swoop.

**Comment by xtianjohnson** [comment]
I have been able to replicate this issue on one of the demo sites - https://site3.demo.microsites.localgovdrupal.org/

You can never get to the menu item encircled in red. 

![item disaapear](https://github.com/user-attachments/assets/8155a548-6e9c-4745-ab2f-4ec0395858c6)


Don't know whether this a bug or but the 'Contact Us' sub  item under the link 'Edward' is never revealed on the homepage when you pass your mouse over 'Edward'

![sub item not displaying](https://github.com/user-attachments/assets/7d586508-a813-498f-a322-d1eb1817fdff)

Let me know if you need any further information.

**Comment by markconroy** [comment]
PR here: https://github.com/localgovdrupal/localgov_microsites_base/pull/268

**Comment by markconroy** [comment]
This should be fixed by https://github.com/localgovdrupal/localgov_microsites_base/pull/268 (updated version)
